eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -31,6 +33,7 @@ parameters:
 
 jobs:
   - job: "Build"
+    timeoutInMinutes: 90
```

Comment:
Does this really need 90 mins?

Reply:
Not anymore :D Brought that down to 24 mins :D

Reply:
Removed timeout